temporallib/client/client.py

```diff
@@ -28,6 +30,13 @@ class Options:
     tls_root_cas: str = None
     auth: AuthOptions = None
 
+async def update_rpc_metadata(client_opt, rpc_metadata, refresh_interval):
+    while True:
```

Comment:
maybe we should put some flag here, that we set to false on the desctructor of the client (\_\_del\_\_). Like that we know we are not trying to update smth that is not being used.

Reply:
Done, thanks!